src/R/Components/Impl/ConnectionManager/Commands/ReconnectCommand.cs

```diff
@@ -21,12 +21,12 @@ public class ReconnectCommand : IAsyncCommand {
             ? CommandStatus.Supported
             : CommandStatus.SupportedAndEnabled;
 
-        public Task<CommandResult> InvokeAsync() {
```

Comment:
You don't want result? It is generally permitted just to call Invoke without actually checking status first.